src/js/paste.js

```diff
@@ -145,7 +145,7 @@ var PasteHandler;
                     }
                 }
             } else {
-                this.pasteHTML(text);
+                this.pasteHTML(text, this.base.options);
```

Comment:
Ah ok.  In that case, can we make a sub-object being something like `options.pasteOptions`?  It would be good if the options passed into here were scoped down to only contain the stuff we care about (instead of the entire options object).

Also, you can just use `this.options` instead of `this.base.options` since the options are stored during initialization.